test/test_asn1.rb

```diff
@@ -566,6 +566,19 @@ def test_decode_constructed_overread
     assert_equal 17, ret[0][6]
   end
 
+  def test_constructive_each
+    data = ['test','data']
+    seq = OpenSSL::ASN1::Sequence.new data
+
+    assert_equal data, seq.entries
+  end
+
+  def test_constructive_nonarray_each
+    seq = OpenSSL::ASN1::Sequence.new 5
+
+    assert_raise(NoMethodError) { seq.each }
```

Comment:
It really hassles but we can't use assert_raise without specifying the exact exception type, because this file is shared with the ruby tree (https://github.com/ruby/ruby/blob/trunk/test/openssl/test_asn1.rb) where the test framework behaves differently. I think not testing is fine this time.